app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardNetworksStep/components/NetworksStepForm/helpers.js

```diff
@@ -42,21 +43,21 @@ export const targetNetworkWithTreeViewAttrs = targetNetwork => ({
 
 export const sourceNetworkWithTreeViewAttrs = (sourceNetwork, selectedCluster) => ({
   ...sourceNetwork,
-  text: sourceNetwork.name,
+  text: `${sourceNetwork.name} \\ ${__('Switch')} ${sourceNetwork.switch_id}`,
```

Comment:
Same as above,  Does this need the context afforded by wrapping the whole text in sprintf?